tests/features/joinup_event/event.edit.feature

```diff
@@ -4,6 +4,24 @@ Feature: "Event page" editing.
   As an owner of the event
   I need to be able to edit it.
 
+  Scenario: Add and remove map
+    Given collections:
+      | title            | logo     | banner     | state     |
+      | Stream of Dreams | logo.png | banner.jpg | validated |
+    And I am logged in as a facilitator of the "Stream of Dreams" collection
+    When I go to the homepage of the "Stream of Dreams" collection
+    And I click "Add event" in the plus button menu
+    When I fill in the following:
+      | Title             | An amazing event                      |
```

Comment:
There is already an event used with this name, description and location in the `add_event.collection` test. Please choose a unique name, description and location.

Reply:
Changes added.